nototools/generate_website_2_data.py

```diff
@@ -554,7 +554,7 @@ def get_css_generic_family(family):
 
 CSS_WEIGHT_MAPPING = {
     'Thin': 250,
```

Comment:
No, we use them to label the font weight with additional information, and to coordinate the samples with the font weight.

Noto Sans CJK DemiLight internally is 350 and is intermediate between Light and Regular, which are 300 and 400 respectively.  There's no existing CSS value for DemiLight, nor is there a name for it.